jupyter_ydoc/yblob.py

```diff
@@ -55,7 +56,7 @@ def get(self) -> bytes:
         :return: Document's content.
         :rtype: bytes
         """
-        return base64.b64decode(self._ysource.get("base64", "").encode())
+        return base64.b64decode(self._ysource["base64"].encode())
```

Comment:
does pycrdt add dynamically a requested key - what would be a bad idea. If not we cannot be sure `base64` exists on `_ysource`.

Reply:
No, pycrdt doesn't create a key on read, and yes you're right I should put back the previous behavior. Thanks, I don't know why I changed it.